audioswitch/src/main/java/com/twilio/audioswitch/bluetooth/BluetoothHeadsetManager.kt

```diff
@@ -50,6 +54,7 @@ internal constructor(
             if (field != value) {
                 field = value
                 logger.d(TAG, "Headset state changed to $field")
+                if (value == Disconnected) enableBluetoothScoJob.cancelBluetoothScoJob()
```

Comment:
This is needed with the new bluetooth intents API because the enableSCOJob triggered from line 134 is still active after getting into the disconnected state.